relatio/utils.py

```diff
@@ -56,7 +56,13 @@ def split_into_sentences(
         time.sleep(1)
         docs = tqdm(docs)
 
+    # Ready to be submitted now and test for merge
     for doc in docs:
+        if not (isinstance(doc["doc"], str)):
```

Comment:
What if the document is a number or a float? Is it replaced by ""?

Reply:
So doc["doc"] is the text field in form of a paragraph. If it is number or a float, it will be meaningless anyway. We can also treat those as blank I guess as they can't be split into sentences.

Reply:
It's important to preserve user input as much as possible. Numbers could be dates, for instance.

Reply:
If the user inputs are numbers in string from, the program works fine. I have a feeling that if the input is a number in int or float form, we'll have the same behavior as NaN, as over there the issue was NaN is float and doesn't have the len() method defined for it.